### PR TITLE
Documentation for compiling on Windows

### LINKED ISSUE #588: Windows Installation Documentation
## Feature

Can we please add a step-by-step document on how to install (and test) GTSAM on Windows?

## Motivation

We already have solid docs for Linux and macos. Let's complete the holy trinity.

This is also the first step to fixing all the warnings and the CI timeouts we see.

## Pitch

Get help from @mikesheffler and @jlblancoc to create a nice little document about installing GTSAM on Windows via all the standard tooling (NuGet, CMake etc).

## Alternatives

NA

## Additional context

NA

### PR DESCRIPTION
**Pull request by varunagrawal**
This fixes #588. Only documentation, no code changes.

@mikesheffler since you are a regular Windows user, I am looking forward to feedback from you.

cc @dellaert

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -70,15 +70,42 @@ execute commands as follows for an out-of-source build:
...
+1. Open Visual Studio.
+2. Select `Open a local folder` and select the GTSAM source directory.
+3. Go to `Project -> CMake Settings`.
+  - (Optional) Set `Configuration name`.
+  - (Optional) Set `Configuration type`.
+  - Set the `Toolset` to `msvc_x64_x64`. If you know what toolset you require, then skip this step.
+  - Update the `Build root` to `${projectDir}\build\${name}`.
+  - You can optionally create a new configuration for a `Release` build.
+  - Set the necessary CMake variables for your use case.
+  - Click on `Show advanved settings`.
```
Comment (mikesheffler) [inline INSTALL.md]: typo (_advanced_)

**Review by mikesheffler — COMMENTED** [review]
@varunagrawal I just noticed this this evening (sorry!). I did a quick scan over it and noticed one typo.

In terms of substantive stuff, I usually used CMake outside of Visual Studio and then just use Visual Studio to build the .sln produced by CMake, so I'd like to actually try to follow the steps as written. I'll try to do it today (Thursday - Friday).

File: INSTALL.md (hunk 1 above)
Comment (varunagrawal) [reply INSTALL.md]: Fixed. Thanks!

**Review by mikesheffler — APPROVED** [review]
Looks good to me. I had to switch to `fix/windows-build` to actually test drive the build, but, I followed the instructions in the docs to generate the .sln using CMake from within Visual Studio.

**Pull request merged**